rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -197,10 +199,14 @@ usb_handler_thread::usb_handler_thread()
 		check_device(0x044F, 0xB660, 0xB660, "Thrustmaster T500 RS Gear Shift");
 
 		// Buzz controllers
-		check_device(0x054C, 0x1000, 0x1040, "buzzer0");
-		check_device(0x054C, 0x0001, 0x0041, "buzzer1");
-		check_device(0x054C, 0x0042, 0x0042, "buzzer2");
-		check_device(0x046D, 0xC220, 0xC220, "buzzer9");
+		if (check_device(0x054C, 0x1000, 0x1040, "buzzer0"))
+			found_buzz = true;
+		if (check_device(0x054C, 0x0001, 0x0041, "buzzer1"))
```

Comment:
else if

Reply:
The way it's set currently is correct, ie if any real buzzer is detected don't emulate.

Reply:
Why? The emulated controlled is now disabled if any real controller is found.
Changing to «else if» wouldn't change this, but it would stop searching for a later models when a real controller is found, making impossible to mix buzzer1 and buzzer2 for example (if it works).
Some Buzz! games support maximum 8 players using 2 controllers.

Reply:
Yes, it's correct as is, I assumed check_device() was read-only, i.e did not register the devices